new-era-commands/index.js

```diff
@@ -1,14 +1,15 @@
 const { globSync } = require('glob');
 const path = require('path');
 
-const commandFiles = globSync('./new-era-commands/**/*.js', { ignore: 'new-era-commands/*.js' });
+const commandFiles = globSync('./new-era-commands/**/*.js', { ignore: ['new-era-commands/*.js', 'new-era-commands/__tests__/**'] });
```

Comment:
Since we moved all our tests from this directory, I think it's safe to revert this change.